Formula/tailscale.rb

```diff
@@ -0,0 +1,34 @@
+class Tailscale < Formula
+  desc "Easiest, most secure way to use WireGuard and 2FA"
+  homepage "https://tailscale.com"
+  url "https://github.com/tailscale/tailscale.git",
+  tag:      "v1.16.2",
+  revision: "b56ba20549aedc0e56d09dca2552ca62421b2cbd"
```

Comment:
```suggestion
      tag:      "v1.16.2",
      revision: "b56ba20549aedc0e56d09dca2552ca62421b2cbd"
```

Reply:
I was surprised that brew style left this, isn't this a rule?